### DISCUSSION TITLE
È possibile inviare automaticamente una fattura una volta creata, tramite Zapier?

### DISCUSSION THREAD

**Discussion by angelarted** [fattureincloud/api#441] [Q&A] [answered]
Buongiorno,
una volta creata la fattura è possibile inviarla automaticamente al cliente? Premetto che, se il cliente non è presente in anagrafica, c'è un altro zap che lo iscrive automaticamente.
Tuttavia non ho trovato nessuna azione tipo "send email" tra le possibilità che Zap offre una volta selezionata l'app di Fattura in Cloud.
Qualcuno si è trovato a dover impostare questa automazione? Come avete risolto?
Grazie,

Simona

**Comment by valmoz** [comment, marked as answer]
Buongiorno @angelarted 
tra le [azioni disponibili su Zapier](https://zapier.com/apps/fatture-in-cloud/integrations) è presente la [Schedule Email](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents/-document_id-/email).

![image](https://github.com/fattureincloud/api/assets/7254293/bfb2a077-c3ef-4745-b87f-2b13e51e7763)

Le consiglio anche di controllare la [guida dedicata](https://developers.fattureincloud.it/docs/guides/invoice-creation#mailbox_with_mail-send-the-document).

grazie
Mauro

**Reply by angelarted** [reply]
Ok grazie, provo a cercare e ti faccio sapere.
Prezioso Mauro!!!!

**Reply by angelarted** [reply]
Risolto grazie! Marco come chiuso.
